lua/astrocommunity/motion/harp-nvim/harp-nvim.lua

```diff
@@ -0,0 +1,25 @@
+return {
+    {
+        "Thorinori/harp-nvim",
+        lazy = false,
```

Comment:
This can be removed.
The plugin should be loaded on the keypresses.

Reply:
I agree

Reply:
```suggestion
```